packages/store/src/internal/state-factory.ts

```diff
@@ -54,21 +51,23 @@ export class StateFactory {
     private _actions: InternalActions,
     private _actionResults: InternalDispatchedActionResults,
     private _stateContextFactory: StateContextFactory,
-    private _internalStateOperations: InternalStateOperations
+    @Optional()
+    @Inject(INITIAL_STATE_TOKEN)
+    private _initialState: any
```

Comment:
`_initialState` - It may be worth renaming it so as not to be confused with the class `InitialState`?

```ts
type InitialValue = any; // alias for more readability

private initialValue: InitialValue;
```

Reply:
I think that this makes it more cryptic.

Reply:
Tweaked the variable name to `initialStateValue` to make it clearer.